.github/workflows/test.yml

```diff
@@ -0,0 +1,46 @@
+name: wake-api-test
+
+on: [push, pull_request]
+
+jobs:
+
+  typecheck:
+    name: 'Wake 0.18.1 typecheck'
+    runs-on: ubuntu-latest
+    env:
+      # $WIT_WORKSPACE path is relative to $GITHUB_WORKSPACE
+      WIT_WORKSPACE: wit-workspace
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: wit-packages/scribble-testsocket-sifie
+        # Fetch all history and tags
+        fetch-depth: 0
+
+    - name: 'Wit Init'
+      uses: sifive/wit/actions/wit@v0.13.1
+      with:
+        command: init
+        arguments: |
+          ${{ env.WIT_WORKSPACE }}
+          -a ./wit-packages/scribble-testsocket-sifie
+        force_github_https: true
+
+    - name: Run wake typecheck
+      working-directory: ${{ env.WIT_WORKSPACE }}
+      run: |
+        set -euxo pipefail
+        # Take back ownership of wit workspace, since it was created while
+        # under a Docker container and permissions default to root.
+        uid=$(id -u)
+        gid=$(id -g)
+        sudo chown -R "$uid:$gid" .
+        # Install Wake
+        curl -L -o /tmp/wake.deb https://github.com/sifive/wake/releases/download/v0.18.1/ubuntu-18-04-wake_0.18.1-1_amd64.deb && \
```

Comment:
I think you meant to say 0.19.0 here.

Reply:
Eh, I guess why not.

Reply:
I was just joking! Glad it worked, though.

Reply:
Haha, yeah, I assumed you were joking, but since this repo is so small and since you just cut the 0.19.0 package today, I figured that I might as well try it out and see what happens.